src/main/java/UStore/RewardsItemGenerator.java

```diff
@@ -5,8 +5,8 @@
     public RewardsItem generateRewardsItem(String itemType){
         RewardsItem rewardsItem= (RewardsItem) rewardsItemFactory.getItem(itemType);
         //TODO: get customer's points earned
-        float customerPoints = 0;
-        rewardsItem.redeemItem(customerPoints);
+        float customerPoints = 12;
```

Comment:
what does this "12" mean here?

Reply:
Oh, it is just a makeup int such that it can runs for now. When TODO is done, I will replace it with the points customer has earned so far.